### ISSUE TITLE
publishSigned succeeds but sonatypeRelease can't find the repository

### ISSUE THREAD

**Issue by raboof** [xerial/sbt-sonatype#35]
publishSigned worked:
```
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1-javadoc.jar.asc
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1.pom.asc
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1-sources.jar
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1.jar.asc
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1.jar
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1-sources.jar.asc
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1-javadoc.jar
[info] 	published ectrace_2.12 to https://oss.sonatype.org/ser
vice/local/staging/deploy/maven2/net/bzzt/ectrace_2.12/0.0.1/e
ctrace_2.12-0.0.1.pom
```

And indeed I see it in the staging repo:

![oss.sonatype.info screenshot](https://cloud.githubusercontent.com/assets/131856/21962167/5b0d1446-db1f-11e6-8abb-748bad013bfe.png)
]

But `sonatypeRelease` (nor `sonatypeList`) finds it:

```
[info] Set current project to ectrace (in build file:/home/aengelen/dev/ectrace/)
[info] Nexus repository URL: https://oss.sonatype.org/service/local
[info] sonatypeProfileName = net.bzzt
[info] Reading staging repository profiles...
[warn] No staging repository is found. Do publishSigned first.
java.lang.IllegalStateException: No staging repository is found. Run publishSigned first
	at xerial.sbt.Sonatype$NexusRESTService.findTargetRepo
sitory(Sonatype.scala:461)
```

I'm probably doing something wrong myself (perhaps around the profileName?), but it's not very clear from the messages what piece I'm missing.

**Comment by raboof** [comment]
Looks like it ended up in the "Central Bundles" staging profile, but when I try to promote that one (by specifying the profile and repository explicitly) it fails because I have no 'promote' permissions for that repo. Looks like I have to add the project using https://issues.sonatype.org/secure/CreateIssue.jspa?issuetype=21&pid=10134 .

**Comment by xerial** [comment]
@raboof Usually staging profiles are named like `netbzzt_1234`, so there must be something wrong the target location.  You can try sonatypeXXX commands to check staging profiles, repositories, etc.

```
$ sbt
> sonatypeList

> sonatypeRelease netbzzt_1234
```

**Comment by raboof** [comment]
@xerial right! indeed after having a staging area created for my organization my repo's ended up there.

Perhaps when no staging repository is found, we should check whether there's a staging profile for this organization in the first place, and if not show a more specific error like "No staging area found for 'net.bzzt'. Have you requested a staging area and successfully published your signed artifact there?".

I'd be happy to create a PR provided you agree with the general idea.

**Comment by xerial** [comment]
This error message would be nice. I'd agree with your idea :)

**Comment by raboof** [comment]
Created #36

**Comment by xerial** [comment]
Fixed in #36

**Comment by raam86** [comment]
If this happens to you make sure your version name does not include `SNAPSHOT`
